cfme/tests/services/test_rest_services.py

```diff
@@ -425,6 +425,49 @@ def _finished(service):
                 delay=5
             )
 
+    @pytest.mark.parametrize(
+        "from_detail", [True, False], ids=["from_detail", "from_collection"]
+    )
+    @test_requirements.rest
+    def test_service_retirement_methods(self, request, appliance, services, from_detail):
+        """Test retiring a service with old method `retire` and new method `request_retire`.
+        Old method is no longer supported and it puts the service into `intializing` state.
+
+        Metadata:
+            test_flag: rest
+
+        Polarion:
+            assignee: pvala
+            casecomponent: Services
+            initialEstimate: 1/4h
+
+        Bugzilla:
+            1698480
+            1713477
+        """
+        service = services[0]
+        if from_detail:
+            service.action.retire()
+        else:
+            appliance.rest_api.collections.services.action.retire(service)
+
+        assert_response(appliance)
+        wait_for(
+            lambda: hasattr(service, "retirement_state"),
+            fail_func=service.reload,
+            num_sec=50,
+            delay=5,
+        )
+        assert service.retirement_state == "initializing"
```

Comment:
Please try to validate Error msg form log as well.

Reply:
Yes, I am currently working on that.

Reply:
so you used matched `matched_patterns` :) it's ok